plutus-scb/src/Plutus/SCB/App.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE DataKinds             #-}
 {-# LANGUAGE DerivingStrategies    #-}
+{-# LANGUAGE ExplicitForAll        #-}
```

Comment:
added but the code doesn't seem to have changed?

Reply:
This will have been necessary at some intermediate step. Removing, thanks. :-)